src/mavedb/lib/validation/dataframe/dataframe.py

```diff
@@ -162,6 +185,31 @@ def validate_dataframe(
         transgenic=score_set_is_sequence_based,
     )
 
+def standardize_dict_keys(d: dict[str, Any]) -> dict[str, Any]:
+    """
+    Standardize the keys of a dictionary by stripping leading and trailing whitespace
+    and removing any quoted strings from the keys.
+
+    Parameters
+    ----------
+    d : dict[str, DatasetColumnMetadata]
+        The dictionary to standardize
+
+    Returns
+    -------
+    dict[str, DatasetColumnMetadata]
+        The standardized dictionary
+    """
+
+    def clean_key(key: str) -> str:
+        key = key.strip()
+        # Only remove quotes if the key is fully quoted
+        if (key.startswith('"') and key.endswith('"')) or (key.startswith("'") and key.endswith("'")):
+            key = key[1:-1]
+
+        return key.strip()
```

Comment:
Could probably extract this into another method `clean_col_name(col: str)` (or similar) and merge it with the function on lines 237-243 so that we have the same logic for the column cleaning in both spots.